### DISCUSSION TITLE
Include an info box(es) about current raids status?

### DISCUSSION THREAD

**Discussion by Cooties-cc** [chromiecraft/chromiecraft#7050] [Suggestions]
Could it be possible to create a central location on the website that you can direct people (or check yourself) that includes all the current nerfs and attunement requirements for raids, as well as which ones are open? I don't think it needs its own dedicated page on the site, maybe just add it to an existing one that's similar in info. This could _greatly_ cut down on anyone asking about nerfs or attunes as a new player, as well as other players not needing to deep search the discord for updates. 

![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/e4d1dbf7-684c-446c-bd5e-ccf88134e8a2)

Something like this simple, but maybe "prettier". Mostly only applies to current raid tiers, as all older raids are released without tunes, and we don't even have a pinky toe into wrath yet. I know there is a rates & changes page, as well as the content prog page, but neither of them explain what the current rates/requirements are for said content. Thought it could be helpful.

**Comment by heyitsbench** [comment]
Very much agree with making this particular piece of info more clear.

**Comment by Cooties-cc** [comment]
Once again asking if there's any chance we could have something like this added to the prog page or something. There's currently zero info circulating as to what raids are tuned to. Attunements don't have to be included anymore, since BT is the last attune which will be dropped after 1 month from release. If it's too much trouble than what it's worth, since come wotlk release I think this info would all be removed, then that's fine. However it would be nice to have some central location for the info regarding raids for people, instead of telling them to go scrub down the announcements in discord.

**Reply by Nyeriah** [reply]
I wouldn’t say there is zero info. Have you checked the pinned messages in the Black Temple channel? Tuning for every raid/instance has been published.

**Reply by heyitsbench** [reply]
|Raids|Status|Tune Status|
|:-:|:-:|:-:|
|Gruul|Released|[Mythic]|
|Magtheridon|Released|[Mythic]|
|Karazhan|Released|25%|
|Serpentshrine Cavern|Released|50%|
|Tempest Keep|Released|50%|
|Mount Hyjal|Released|75%|
|Black Temple|Released|[Mythic]|
|Zul'Aman|Unreleased|N/A|
|Sunwell Plateau|Unreleased|N/A|

Current tuning as far as I understand it. I do still agree with this suggestion, just putting the current status here for anyone who wants to see at a glance.
